DataFormats/TrackerRecHit2D/interface/SiPixelRecHitQuality.h

```diff
@@ -124,7 +124,7 @@ class SiPixelRecHitQuality {
       if (prob > 1.0f && prob <= 1.0f + std::numeric_limits<float>::epsilon()) {
         prob = 1;
       } else if (prob < 0.0f || prob > 1.0f + std::numeric_limits<float>::epsilon()) {
-        warningOutOfBoundProb("Q", prob, qualWord);
+        //warningOutOfBoundProb("Q", prob, qualWord);
```

Comment:
As per previous statement (cf https://github.com/cms-sw/cmssw/pull/38967#discussion_r937899869) commented code is not advised in the reco stack.
If you are going to keep it, then at least put a comment on why it is so.

Reply:
If there's no objection, I can simply delete this line. I kept it commented out in case others think it may be useful to keep. If not, I will delete it.

Reply:
I would guess that the default value should not get a warning, but perhaps everything else still should.
@cms-sw/trk-dpg-l2  please advise

Reply:
> I would guess that the default value should not get a warning, but perhaps everything else still should. @cms-sw/trk-dpg-l2 please advise

I guess my tag went only to the DPG conveners
@ferencek @mroguljic 
(please clarify also if there is a reco contact replacing Oz)

Reply:
We've been looking (unsuccessfully) for a reco contact for more than a year now. 

Regarding the proposed code changes, simply commenting out or deleting a call to `warningOutOfBoundProb` does not look like a good solution to me as it changes the legacy behavior of this code. Possible solutions:
1) As Slava suggested, modify the code so that the default value does not cause a warning
2) Add some sort of verbosity flag which would allow suppressing warnings when the Q probability is not computed.

Reply:
@ferencek, this is addressed in [https://github.com/cms-sw/cmssw/pull/39776/commits/8b4f5df85d4bd958f1b9a188bc5f3d9ef6e6925b](8b4f5df).
Only the default value does not cause the warning, which otherwise is still there and active.